examples/empty/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "conditional-service"
```

```suggestion
name = "prog-without-svcs"
```
including folder name

---

renamed